pkg/api/v1/types.go

```diff
@@ -988,6 +990,12 @@ type Container struct {
 	// Whether this container should allocate a TTY for itself, also requires 'stdin' to be true.
 	// Default is false.
 	TTY bool `json:"tty,omitempty"`
+	// Whether this container should send stdout.
```

Comment:
Note that we already have the ability to attach to running containers.

https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/attach.go